### DISCUSSION TITLE
Multimarkdown Tables and others are disabled

### DISCUSSION THREAD

**Discussion by Papillone** [requarks/wiki#3255] [Error / Bug Report]
Hello,

I can't use some features like Multimarkdown Tables, Diagrams, Mathjax ...
I have activeted this modules in the configuuration under "Rendering -> Markdown->html" but the buttons are disabled in the editor? What is going wrong?

See the screenshot: ![https://bilder-teilen.de/image/pdflH](/https://bilder-teilen.de/image/pdflH)

OS: Linux - Debian (buster) 10 x64
Wiki.js version: 2.5.170
Database engine: postgres 11.9

**Comment by leomarques3** [comment]
I'm with the same issue, did you fixed it?

**Comment by Papillone** [comment]
> 
> 
> I'm with the same issue, did you fixed it?

Unfortunately I have no solution.

**Comment by tchapi** [comment]
Still an issue here, anyone has a solution ?

**Comment by juliuspopp** [comment]
same problem here, 5/8 buttons disabled in Firefox.
Ok, according to this issue it is not a bug but a missing feature: https://github.com/Requarks/wiki/issues/2637
